packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared-settings.html

```diff
@@ -103,7 +103,7 @@ <h4 class="oui-box__heading" data-translate="hosting_cdn_shared_option_http_geol
              data-ng-init="prefetch = $ctrl.getOption($ctrl.model.optionTypes.PERF, $ctrl.cdnOptionTypeEnum.PREFETCH).api">
 
             <h4 class="oui-box__heading" data-translate="hosting_cdn_shared_option_prefetch_title"></h4>
-            <p data-translate="hosting_cdn_shared_option_prefetch_info"></p>
+            <span data-translate="hosting_cdn_shared_option_prefetch_info"></span>
```

Comment:
**question:** Why not prefer wrapping the `<oui-switch>` element into a `<oui-field>`?
see: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-switch-webcomponent--validation

Reply:
Oui-field had a large bottom padding which we can't managed.